packages/salesforce-adapter/src/change_validators/last_layout_removal.ts

```diff
@@ -0,0 +1,101 @@
+/*
+*                      Copyright 2023 Salto Labs Ltd.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with
+* the License.  You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+import {
+  ChangeError,
+  ChangeValidator,
+  getChangeData,
+  InstanceElement,
+  isInstanceChange, isInstanceElement, isObjectTypeChange,
+  isRemovalChange,
+} from '@salto-io/adapter-api'
+import { collections } from '@salto-io/lowerdash'
+import _ from 'lodash'
+import { apiName } from '../transformers/transformer'
+import { INSTANCE_FULL_NAME_FIELD, LAYOUT_TYPE_ID_METADATA_TYPE as LAYOUT_METADATA_TYPE } from '../constants'
```

Comment:
Nit: Why this rename?
```suggestion
import { INSTANCE_FULL_NAME_FIELD, LAYOUT_TYPE_ID_METADATA_TYPE } from '../constants'
```